### PR TITLE
Add combine sets & Revert to Simple structure

### PR DESCRIPTION
**Pull request by cmharlow**
What this PR does:
- adds DAGs for DPLAH, Temple, Historic Pittsburgh
- reverts to "simple" DAG setup
- does some slack post work so we can pass in Connection ID (for eventually using different Slack Conn)

### CODE REVIEW

File: funcake_dplah_dag.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+)
+
+# Define the single task in this controller DAG
+OAI_TRIGGER = TriggerDagRunOperator(
+    task_id="oai_trigger",
+    trigger_dag_id="funcake_generic_oai",
+    python_callable=conditionally_trigger,
+    params={"condition_param": True,
+            "message": "Triggering DPLAH OAI DAG",
+            "test": "test",
```
Comment (sensei100) [inline funcake_dplah_dag.py]: @cmharlow This is leftover from debugging.  We should take it out.

File: funcake_dplah_dag.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            "OAI_CONFIG": OAI_CONFIG,
+            "OAI_MD_PREFIX": MD_PREFIX,
+            "OAI_INCLUDED_SETS": OAI_INCLUDED_SETS,
+            "OAI_ENDPOINT": OAI_ENDPOINT,
+            "EXCLUDE_SETS": OAI_EXCLUDED_SETS,
```
Comment (sensei100) [inline funcake_dplah_dag.py]: This was changed to OAI_EXCLUDED_SETS in the target dag.

File: funcake_dplah_dag.py (hunk 1 above)
Comment (cmharlow) [reply funcake_dplah_dag.py]: noted that, putting in different branch

File: funcake_dplah_dag.py (hunk 2 above)
Comment (cmharlow) [reply funcake_dplah_dag.py]: good catch; trying to keep myself consistent

**Pull request merged**